LTI-EJV/prompts.md

```
@@ -0,0 +1,261 @@
+# Prompts Utilizados para el Proyecto LTI-EJV
+
+## Metaprompting
+
+Sigo asiendo uso de metaprompting ya que me ha dado buenos resultados. Usé el siguiente prompt cen Claude.ai, adjuntando los documentos 🎥 Primeros pasos en un proyecto de software.docx y ReadMe.md.
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)